src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  public shape: 'triangle' = 'triangle';
```

Comment:
Consider creating an enum for the shape 
```suggestion
  public shape: Shape = Shape.triangle;
```